### PR TITLE
update engine version. make sure callFunc can pass 0 valued  argument.

### PR DESCRIPTION
**Pull request by nantas**
> Makes sure these boxes are checked before submitting your PR - thank you!
> - [x] Check that your javascript is following our [style guide](https://github.com/cocos-creator/fireball/blob/dev/.github/CONTRIBUTING.md) and end files with a newline
> - [x] Document new code with comments in source code based on [API Docs](https://github.com/cocos-creator/fireball#api-docs)
> - [x] If your pull request has gone "stale", you should **rebase** your work on top of the latest version of the upstream branch.
> - [x] If your commit history is full of small, unimportant commits (such as "fix pep8" or "update tests"), **squash** your commits down to a few, or one, discreet changesets before submitting a pull request.

@cocos-creator/engine-admins

### CODE REVIEW

File: cocos2d/actions/CCActionInstant.js [hunk 1]
```diff
@@ -440,7 +440,7 @@ cc.CallFunc = cc.ActionInstant.extend({
         if (selectorTarget) {
             this._selectorTarget = selectorTarget;
         }
-        if (data) {
+        if (data || data === 0) {
```
Comment (jareguo) [inline cocos2d/actions/CCActionInstant.js]: 是否也应该支持空字符串？

Comment (jareguo) [inline cocos2d/actions/CCActionInstant.js]: @pandamicro 麻烦看下

**Comment by nantas** [comment]
后面更新的是 crossLink 的修复，和题目无关

File: cocos2d/actions/CCActionInstant.js (hunk 1 above)
Comment (pandamicro) [inline cocos2d/actions/CCActionInstant.js]: 判断改成 
if (data !== undefined)

**Pull request merged**